### PR TITLE
ENH: antsIntroduction handles RA and RI transformations

### PR DESCRIPTION
**Pull request by lsqshr**
when generating the _list_outputs if -t RA or -t RI is set, *_warp_field
outputs will not be required

### CODE REVIEW

File: nipype/interfaces/ants/legacy.py [hunk 1]
```diff
@@ -94,16 +94,20 @@ class antsIntroduction(ANTSCommand):
 
     def _list_outputs(self):
         outputs = self._outputs().get()
+        transmodel = self.inputs.transformation_model
+
+        # When transform is set as 'RI'/'RA', no wrap fields will be outputed
+        if isdefined(transmodel) and transmodel not in ['RI', 'RA']:
```
Comment (satra) [inline nipype/interfaces/ants/legacy.py]: if someone unsets/undefines this, what will antsIntroduction do? do you know what the ants default behavior is?

Comment (lsqshr) [inline nipype/interfaces/ants/legacy.py]: Hi @satra, thanks for reviewing. I just double-checked, if transmodel is undefined, antsIntroduction will set '-t  GR' which is 'greedy SyN' by default.

Comment (satra) [inline nipype/interfaces/ants/legacy.py]: @lsqshr -  in that case the if statement should perhaps be:

`if not isdefined(transmodel) or (isdefined(transmodel) and transmodel not in ['RI', 'RA']):`

Comment (lsqshr) [inline nipype/interfaces/ants/legacy.py]: @satra Thx for pointing out. You are right about it. I just made the according change and locally tested it with real nii images.

**Pull request merged**